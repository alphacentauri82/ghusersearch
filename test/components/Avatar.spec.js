import React from 'react';
import renderer from 'react-test-renderer';
import {
  StyleSheetTestUtils,
} from 'aphrodite';
import Avatar from 'components/Avatar';

describe('Component: Avatar', () => {

  beforeAll(() => {
    StyleSheetTestUtils.suppressStyleInjection();
  });

  afterAll(() => {
    StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
  });

  it('should render', () => {
    const component = renderer.create(
      <Avatar
        url="http://avatar.net"
        name="Simon"
        width={300}
      />
    );
    expect(component).toMatchSnapshot();
  });

});
