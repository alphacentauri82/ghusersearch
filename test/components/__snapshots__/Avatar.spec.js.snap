// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Avatar should render 1`] = `
<div
  className="Image_e296pg"
  style={
    Object {
      "minHeight": 60,
    }
  }
>
  <img
    alt="Simon's avatar"
    className="Avatar_1pcpsid u-imgResponsive"
    onLoad={[Function]}
    src="http://avatar.net&s=600"
    width={300}
  />
  <svg />
</div>
`;
